ingestion/src/metadata/workflow/workflow_output_handler.py

```diff
@@ -141,15 +138,18 @@ def _print_summary(self, steps: List[Step]):
                 log_ansi_encoded_string(message=f"Filtered: {step_summary.filtered}")
 
             log_ansi_encoded_string(message=f"Errors: {step_summary.errors}")
+            log_ansi_encoded_string(
+                message=f"Success %: {step.get_status().calculate_success()}"
+            )
 
         self._print_failures_if_apply(failures)
 
-        total_success = max(total_records, 1)
+        # If nothing is processed, we'll have a success of 100%
+        success_pct = mean([step.get_status().calculate_success() for step in steps])
```

Comment:
this will fail with this error if steps is empty. 
```
statistics.StatisticsError: mean requires at least one data point
```

Not sure if we currently have such a use-case but worth handling this just in case and ask the users to check config or send them to slack.

Reply:
should not happen since we pass the steps by hand but fair enough, will handle that